kroxylicious-systemtests/src/main/resources/helm_strimzi_overrides.yaml

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# Helm Overrides File for Strimzi used by the system tests.
+replicas: 1
+featureGates: "+UseKRaft"
```

Comment:
why do we need this line?  Isn't KRaft being enabled the default https://github.com/strimzi/strimzi-kafka-operator/blob/main/CHANGELOG.md#0400